pysc2/lib/features.py

```diff
@@ -211,17 +212,41 @@ def __init__(self, game_info=None, screen_size_px=None, minimap_size_px=None,
     Raises:
       ValueError: if game_info is None and screen or minimap sizes are missing.
     """
+    self._map_size = None
     if game_info:
       fl_opts = game_info.options.feature_layer
       screen_size_px = point.Point.build(fl_opts.resolution)
       minimap_size_px = point.Point.build(fl_opts.minimap_resolution)
+      self._map_size = point.Point.build(game_info.start_raw.map_size)
```

Comment:
Hmm, this will make things more challenging, in that if we don't have a game_info, as we don't in our agents, you can't get that observation.